### PR TITLE
Enabled support for children above EditorContent

### LINKED ISSUE #672: Add support for child components to Editor
- [ ]  Add support for accepting children above EditorContent

### PR DESCRIPTION
**Pull request by gaagul**
- Fixes #672 

**Description**

- Added: support for children above EditorContent

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**
@AbhayVAshokan _a

### CODE REVIEW

**Comment by gaagul** [comment]
@AbhayVAshokan _a I will work on the documentation once all the requirements are completed.

**Comment by gaagul** [comment]
@AbhayVAshokan I have added attachmentsClassname prop, Please review.

**Comment by gaagul** [comment]
@AbhayVAshokan _a

File: src/components/Editor/index.jsx [hunk 1]
```diff
@@ -192,7 +198,7 @@ const Editor = (
           {isAttachmentsActive && (
             <Attachments
               attachments={attachments}
-              className="ne-attachments--integrated"
+              className={`ne-attachments--integrated ${attachmentsClassName}`}
```
Comment (AbhayVAshokan) [inline src/components/Editor/index.jsx:201]: Use `classnames` package to use the second class if it is present. neetoEditor has similar usages you may refer to.

**Review by AbhayVAshokan — APPROVED** [review]
@gaagul _a please merge after making this change.

**Pull request merged**